schema_based_on_diagram.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE patients (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(255),
+    date_of_birth DATE,
+    PRIMARY KEY (id)
+);
+
```

Comment:
- [ ] According to the project requirements, all Foreign Keys need to have indexes created:

```sql
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
CREATE INDEX ON invoice_items (invoice_id);
CREATE INDEX ON invoice_items (treatment_id);
CREATE INDEX ON medical_histories_has_treatments (medical_history_id);
CREATE INDEX ON medical_histories_has_treatments (treatment_id);
```

Please, add them so the database's performance improves! 💪 

- [ ] All tables need a primary key set, just as you did in the `patients` table.

Reply:
Thanks a lot for pointing this out. I will add these.